code/modules/reagents/chemistry/recipes.dm

```diff
@@ -58,6 +58,10 @@
 	///Tagging vars
 	///A bitflag var for tagging reagents for the reagent loopup functon
 	var/reaction_tags = NONE
+	var/centrifuge_recipe = FALSE
+	var/pressure_required = 0
+	var/radioactivity_required = 0
```

Comment:
should be:
```suggestion
	var/required_pressure = 0
	var/required_radioactivity = 0
```
to match required_temp, also shouldn't it be radiation and not radioactivity?